src/modules/form/components/group/DisabilityTable.tsx

```diff
@@ -52,7 +52,7 @@ const DisabilityTable = ({
       ?.linkId;
   }, [item]);
 
```

Comment:
Not sure where this comment will belong, but placing it here. There is a difference in something between QA and this branch:

* Select Yes to Physical Disability
* Select Yes to Physical Disability is a Disabling Condition
* Clear out the value for Physical Disability
  * **QA**: this results in "Physical Disability is a Disabling Condition" being cleared out as well
  * **This Branch**:  "Physical Disability is a Disabling Condition" appears to remain as Yes.

Pretty sure this doesn't stem from logic within the DisabilityTable component (which mostly deals with the "overall disabling condition" at the bottom), but rather something about "autofill dependencies" . Let me know if you'd like me to spend some more time tracking it down or pairing!

Reply:
good find! I've patched this behavior in DependentFormItemWrapper